book/website/project-design/data-governance.md

```diff
@@ -0,0 +1,30 @@
+(pd-dg)=
+
+# Data Governance
+
+(pd-dg-prerequisites)=
+## Prerequisites
+
+| Prerequisite | Importance | 
+| -------------|----------|
+| {ref}`Research Data Management <rr-rdm>` | Helpful |
+
+(pd-dg-summary)=
+## Summary
+
+As researchers and data practitioners, we need to consider how to make decisions as individuals and on teams for how we collect, manage, apply, and grant access to the datasets we work with.
+
+In this chapter, we give an overview of data governance best practices for different contexts such as machine learning. 
```

Comment:
```suggestion
In this chapter, we give an overview of data governance best practices for different contexts such as machine learning. 
```

Are there any other contexts?

Reply:
Hey @HaoZeke! Was there a change you'd suggest here - seems like the suggestion somehow doesn't include any changes?

Reply:
Ah I meant to highlight that there were no other contexts discussed, so in that case it would be best practices for machine learning.

Reply:
Yup good flag! hopefully there will be more in the future, ML is my domain so it just happens to be the first chapter :)